dev-util/clue/clue-2.2.0.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="A GUI tracing tool"
+DESCRIPTION="Clue is a comprehensive GUI tracing tool, allowing inspection on almost all of the haiku native objects."
+HOMEPAGE="https://github.com/HaikuArchives/Clue"
+COPYRIGHT="Jeff Braun and Mathew Hounsell"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/HaikuArchives/Clue/archive/master.tar.gz"
+CHECKSUM_SHA256="e17c9ed26ea726e9ca7590c41187c17ee19007064529b290aff6eb2df8ed1e29"
+SOURCE_DIR="Clue-master"
```

Comment:
Didn't check this but maybe?
```diff
-SOURCE_DIR="Clue-master"
+SOURCE_DIR="Clue-master/source/library"
```
This would remove the cd into the BUILD section

Reply:
@Begasus: Yep, this would work.

@mpcs1: However, since we cannot use `master.tar.gz`, we have to use a srcGitRev. (That's because master may change, and we want a specific revision to make sure we always get the same tarball.)
So we should have:
```diff
 REVISION="1"
+srcGitRev="0d49930871aa9e3c539f094f299d8a1f43150303"
-SOURCE_URI="https://github.com/HaikuArchives/Clue/archive/master.tar.gz"
+SOURCE_URI="https://github.com/HaikuArchives/Clue/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="e17c9ed26ea726e9ca7590c41187c17ee19007064529b290aff6eb2df8ed1e29"
+CHECKSUM_SHA256="33fa1f1000c62090ed285bc8e9e81f2f1bccc50a0ef2ab4ccaa1c4949f5dcfc1"
+SOURCE_FILENAME="Clue-$portVersion-$srcGitRev.tar.gz"
-SOURCE_DIR="Clue-master"
+SOURCE_DIR="Clue-$srcGitRev/source/library"
-PATCHES="clue-2.2.0.patchset"
+PATCHES="clue-$portVersion.patchset"
```